docs/reference.rst

```diff
@@ -135,6 +135,10 @@ Meta options
 
         .. versionadded: 2.6.0
 
+        Note that if a keyword argument specified in the :attr:`rename` dict
+        is not explicitly passed when instantiating, it will simply be ignored
+        and the key will not be present.
```

Comment:
This behavior was my initial expectation, I am not sure if it's worth documenting it. What do you think?

IMHO, a changelog entry should be added for this change (in `docs/changelog.rst`).

Reply:
Agreed that it probably does not make sense in the docs. How would you like me to add the changelog entry? I would assume it would not be added until a release is cut. Should I just go ahead and add it to the `2.11.2` release?

Reply:
I haven't looked at what we're using in `FactoryBoy`, but typically unreleased features go under a *Master* or similar heading to keep from promising what the next version number will be. And then when a release is cut, *Master* gets updated to *2.11* or whatever...

Reply:
I believe the `(unreleased)` mention makes the distinction in `FactoryBoy`, so I would say to add the changelog entry in the `2.11.2 (unreleased)` section.
https://github.com/FactoryBoy/factory_boy/blob/master/docs/changelog.rst#2112-unreleased

What do you think?

Reply:
Not something I feel strongly on. I think hardcoding a version doesn't make sense, since we've tried to follow semvar, so typically don't know for sure until a version is cut whether it will be `2.11.2` / `2.12` / `3.0-beta` etc. But again, not a big deal if this is how it's already being done. @rbarrois any thoughts on your end?

Reply:
For the record, I will rebase and include this change under the 2.11.2 (unreleased) changelog entry, as a bugfix.